.github/workflows/soak-testing.yml

```diff
@@ -18,12 +18,12 @@ on:
     - cron: '0 */24 * * *'
 env:
   # NOTE: The configuration of `APP_PROCESS_EXECUTABLE_NAME` is repo dependent
-  APP_PROCESS_EXECUTABLE_NAME: python3
+  APP_PROCESS_EXECUTABLE_NAME: dotnet
   AWS_DEFAULT_REGION: us-east-1
   DEFAULT_TEST_DURATION_MINUTES: 300
   HOSTMETRICS_INTERVAL_SECS: 600
-  CPU_LOAD_THRESHOLD: 71
-  TOTAL_MEMORY_THRESHOLD: 2147483648
+  CPU_LOAD_THRESHOLD: 44
```

Comment:
Just curious, are these two values evaluated based on the the actual running?

Reply:
Yes as seen in the screenshots above and below! The thresholds are around these values, so I felt these were good choices for the Soak Tests on .NET.